frontend/amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -152,6 +157,10 @@ export class TableDetail extends React.Component<
   state = {
     sortedBy: SORT_CRITERIAS.sort_order,
     currentTab: this.getDefaultTab(),
+    isRightPanelOpen: false,
+    isRightPanelPreExpanded: false,
```

Comment:
Open and Expanded? are they different?

Reply:
I have the "preexpanded" var that is only set once, if someone accesses the page based on the link to a specific column then it would expand the panel and set this var to true so it knew it was expanded and wouldn't keep updating the state with future renders. and then the regular "is open" var is used to keep track for the regular toggling of the panel